engine/apps/grafana_plugin/helpers/client.py

```diff
@@ -274,6 +274,21 @@ def get_alerting_notifiers(self):
     def get_grafana_plugin_settings(self, recipient: str) -> APIClientResponse:
         return self.api_get(f"api/plugins/{recipient}/settings")
 
+    def get_service_account(self, login: str) -> APIClientResponse:
+        return self.api_get(f"api/serviceaccounts/search?query={login}")
+
+    def create_service_account(self, name, role) -> APIClientResponse:
+        return self.api_post("api/serviceaccounts", {"name": name, "role": role})
+
+    def create_service_account_token(self, service_account_id, name, seconds_to_live=None):
+        token_config = {"name": name}
+        if seconds_to_live:
+            token_config["secondsToLive"] = seconds_to_live
+        return self.api_post(f"api/serviceaccounts/{service_account_id}/tokens", token_config)
+
+    def get_service_account_token_permissions(self):
+        return self.api_get("api/access-control/user/permissions")
```

Comment:
if you're feeling ambitious you could update the return types similar to [this](https://github.com/grafana/oncall/pull/3189/files#diff-910e9ef3a0f314a531500237fae412769bc6e4b9400ff604192d2a61f9336ec6R243) to strongly type the return payload